StyleCop.Analyzers/StyleCop.Analyzers/Helpers/RobustFixAllProvider.cs

```diff
@@ -9,9 +10,24 @@
     using Microsoft.CodeAnalysis.CSharp.Syntax;
     using SpacingRules;
 
-    internal sealed class SA1407SA1408FixAllProvider : FixAllProvider
+    internal sealed class RobustFixAllProvider : FixAllProvider
```

Comment:
I lean towards calling this `DocumentBasedFixAllProvider` instead, indicating it is a `FixAllProvider` that applies changes to one `Document` at a time.